highlights for 2023-01-19 (#1678)
giantswarm/docs: src/content/changes/highlights/2023-01-19.md

---
date: 2023-01-19T11:00:00
title: Highlights for the week ending January 19, 2021
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

> ## Apps
> 
> ## User interfaces

Reviewer: ```suggestion
## Apps

- [falco-app](https://github.com/giantswarm/falco-app) version [v0.4.3](https://github.com/giantswarm/falco-app/blob/main/CHANGELOG.md#043---2022-12-21) uses Falco's much slimmer `no-driver` image, reducing the size and dependencies contained in the image.
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.13.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0131---2022-12-21) updates to Kyverno version 1.8.4 and policy-reporter version 2.11.0, bringing considerable resource utilization improvements. This version also excludes the `kube-system` namespace from Kyverno webhooks by default. `kube-system` reports can be re-enabled in the app values.
- [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.7.0](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#070---2023-01-11) introduces horizontal autoscaling based on Prometheus scrape times.
- [security-pack](https://github.com/giantswarm/security-pack) version [0.11.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#0110---2023-01-11) includes new versions of all pack components and makes several adjustments to installed apps and configuration. Please refer to the changelog for details.
- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.3.2](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#032---2022-12-21) updates to Trivy operator 0.7.1, aligns some app configuration with new official charts, and disables the secret scanner by default. Exposed secret reports can be re-enabled in the app values.

## User interfaces
```

---
date: 2023-01-19T11:00:00
title: Highlights for the week ending January 19, 2021
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

> With [kubectl-gs](https://github.com/giantswarm/kubectl-gs) release version [v2.31.0](https://github.com/giantswarm/kubectl-gs/blob/main/CHANGELOG.md#2310---2023-01-18) and [happa](https://github.com/giantswarm/happa) release version [v1.57.0](https://github.com/giantswarm/happa/releases/tag/v1.57.0) support for multiple identity provider got added. This means that you can connect multiple identity provider at the same time. Please reach out to us if you need support to implement this feature.

Author: ```suggestion
With [kubectl-gs](https://github.com/giantswarm/kubectl-gs) release version [v2.31.0](https://github.com/giantswarm/kubectl-gs/blob/main/CHANGELOG.md#2310---2023-01-18) and [happa](https://github.com/giantswarm/happa) release version [v1.57.0](https://github.com/giantswarm/happa/releases/tag/v1.57.0) support for multiple identity provider got added. This means that you can reduce the risk of failure or admit users from different identity providers by connecting multiple identity providers at the same time. Please reach out to us if you need support to implement this feature.
```

---
date: 2023-01-19T11:00:00
title: Highlights for the week ending January 19, 2021
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General
## Apps

- [falco-app](https://github.com/giantswarm/falco-app) version [v0.4.3](https://github.com/giantswarm/falco-app/blob/main/CHANGELOG.md#043---2022-12-21) uses Falco's much slimmer `no-driver` image, reducing the size and dependencies contained in the image.

> - [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.13.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0131---2022-12-21) updates to Kyverno version 1.8.4 and policy-reporter version 2.11.0, bringing considerable resource utilization improvements. This version also excludes the `kube-system` namespace from Kyverno webhooks by default. `kube-system` reports can be re-enabled in the app values.
> - [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.7.0](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#070---2023-01-11) introduces horizontal autoscaling based on Prometheus scrape times.
> - [security-pack](https://github.com/giantswarm/security-pack) version [0.11.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#0110---2023-01-11) includes new versions of all pack components and makes several adjustments to installed apps and configuration. Please refer to the changelog for details.
> - [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.3.2](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#032---2022-12-21) updates to Trivy operator 0.7.1, aligns some app configuration with new official charts, and disables the secret scanner by default. Exposed secret reports can be re-enabled in the app values.

Reviewer: ```suggestion
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.13.1](https://github.com/giantswarm/kyverno-app/blob/main/CHANGELOG.md#0131---2022-12-21) updates to Kyverno version 1.8.4 and policy-reporter version 2.11.0, bringing considerable resource utilization improvements. This version also excludes the `kube-system` namespace from Kyverno webhooks by default. `kube-system` reports can be re-enabled in the app values.
- [security-pack](https://github.com/giantswarm/security-pack) version [0.11.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#0110---2023-01-11) includes new versions of all pack components and makes several adjustments to installed apps and configuration. Please refer to the changelog for details.
- [starboard-exporter](https://github.com/giantswarm/starboard-exporter) version [v0.7.0](https://github.com/giantswarm/starboard-exporter/blob/main/CHANGELOG.md#070---2023-01-11) introduces horizontal autoscaling based on Prometheus scrape times.
- [trivy-operator-app](https://github.com/giantswarm/trivy-operator-app) version [0.3.2](https://github.com/giantswarm/trivy-operator-app/blob/main/CHANGELOG.md#032---2022-12-21) updates to Trivy operator 0.7.1, aligns some app configuration with new official charts, and disables the secret scanner by default. Exposed secret reports can be re-enabled in the app values.
```

---
date: 2023-01-19T11:00:00
title: Highlights for the week ending January 19, 2021
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## General

With [kubectl-gs](https://github.com/giantswarm/kubectl-gs) release version [v2.31.0](https://github.com/giantswarm/kubectl-gs/blob/main/CHANGELOG.md#2310---2023-01-18) and [happa](https://github.com/giantswarm/happa) release version [v1.57.0](https://github.com/giantswarm/happa/releases/tag/v1.57.0) support for multiple identity provider got added. This means that you can connect multiple identity provider at the same time. Please reach out to us if you need support to implement this feature.

> ## Apps

Author: ```suggestion

## Apps
```
